EnvButtonStartingPosition.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  EnvButtonStartingPosition.swift
+//  EnvChanger
+//
+//  Created by Nikola B Nikolov on 6.12.21.
+//
+
+import UIKit
+
+/// Static values representing positions along an axis.
+private extension CGFloat {
+    static let start: CGFloat = 0.1
+    static let middle: CGFloat = 0.5
+    static let end: CGFloat = 0.9
+}
+
+/// Struct describing an `EnvButtonStartingPosition`.
+public struct EnvButtonStartingPosition {
+    
+    /// Enum defining possible vertical positions.
+    public enum VerticalPosition {
```

Comment:
[Optional]
I am not a fan of local object definitions being public. If you need to access it from outside, better to declare it outside of the scope of the struct. The main advantage of local declarations is not cluttering the namespace in my opinion, but you get some indentation as a result.

Reply:
Moved enums out of struct.